RAT_README.txt

```diff
@@ -60,6 +60,7 @@ $MY_SQROOT/../sql/bin/SqlciErrors.txt
 
 dcs/conf/servers 
 dcs/conf/backup-masters 
+dcs/conf/master
                  :  Apache Trafodion DCS default configuration file
                  -> configuration file do not handle comments
 
```

Comment:
This is not documentation but a short description for each component  for list of files that excludes the copyright information as mentioned in the initial section of the this file.